hikari/impl/rest.py

```diff
@@ -3969,6 +3970,9 @@ def interaction_message_builder(
     def interaction_modal_builder(self, title: str, custom_id: str) -> special_endpoints.InteractionModalBuilder:
         return special_endpoints_impl.InteractionModalBuilder(title=title, custom_id=custom_id)
 
+    def interaction_premium_required_builder(self) -> special_endpoints.InteractionPremiumRequiredBuilder:
```

Comment:
This is not needed

Reply:
Why? Every other builder has a rest method (for some reason, anyways).

Reply:
I miss read it, nvm.

Also, the reason for it is RESTBot.